### ISSUE TITLE
Pretrained network weight for S3DIS does not work

### ISSUE THREAD

**Issue by nakhan51** [isl-org/Open3D-ML#364]
Hi, when I used the pre-trained network weight for both KPFCNN and RANDLAnet, I am getting a size mismatch error for the current model. So, the model I loaded and the target model is not identical, Does any of you know how to solve this?

RuntimeError: Error(s) in loading state_dict for KPFCNN:
	size mismatch for encoder_blocks.0.KPConv.weights: copying a param with shape torch.Size([15, 5, 64]) from checkpoint, the shape in current model is torch.Size([15, 2, 64]).
	size mismatch for head_softmax.mlp.weight: copying a param with shape torch.Size([13, 128]) from checkpoint, the shape in current model is torch.Size([19, 128]).
	size mismatch for head_softmax.batch_norm.bias: copying a param with shape torch.Size([13]) from checkpoint, the shape in current model is torch.Size([19]).

**Comment by sanskar107** [comment]
@nakhan51 I cannot reproduce this problem with S3DIS dataset. Could you tell which dataset you are using and the config file?

**Comment by sanskar107** [comment]
Closing due to inactivity. Feel free to reopen if the issue persists.

**Comment by BastianGit** [comment]
Hi, I have the same issue but using the semantickitty dataset.

-  I have the semantickitty dataset downloaded locally.
-  I follow the pre-defined script of "Running a pretrained model for semantic segmentation" at https://github.com/isl-org/Open3D-ML/blob/master/README.md#using-predefined-scripts.
-  I replaced the model weights download link with the correct one in the "readme" in the "dev" branch.
-  I am also using the "randlanet_semantickitty.yml" as configuration file found in the configs folder in the dev branch. 
- I replaced the checkpoint path name with the correct one in variable "ckpt_path" in the code.

Everything seems to have to work but for some reason the pre-trained weights does not match the config file "randlanet_semantickitty.yml". I just want to run inference on the semantickitty dataset, no training, just segmenting. Any ideas how I can solve this error?
![Error_size_mismatch](https://user-images.githubusercontent.com/64791224/152219272-d7cf0a16-b2cb-420e-b42c-868a5d82bb3c.PNG)
